src/pymoca/ast.py

```diff
@@ -839,4 +907,4 @@ def update_parent_refs(self) -> None:
         self._update_parent_refs(self)
 
     def __repr__(self):
-        return '{}(classes={!r})'.format(type(self).__name__, self.classes)
+        return "{}(classes={!r})".format(type(self).__name__, self.classes)
```

Comment:
I was concerned black migh mess up `__repr__`'s in the ast that should use single quotes, but it appears all is well. In hindsight, I think even if I had not used `!r` everywhere, black would respect single quotes inside double quotes.